HW8/OlenaBudko/Task_8_3/main.py

```
@@ -0,0 +1,22 @@
+from utils import *
```

Ruff (0.8.2)

`from utils import *` used; unable to detect undefined names

(F403)